lib/backend/manager.go

```diff
@@ -139,3 +144,18 @@ func (m *Manager) GetClient(namespace string) (Client, error) {
 	}
 	return nil, ErrNamespaceNotFound
 }
+
+// IsReady returns whether the backends are ready (reachable).
+// A backend must be explicitly configured as required for readiness to be checked.
+func (m *Manager) IsReady() (bool, error) {
+	for _, b := range m.backends {
+		if !b.mustReady {
+			continue
+		}
+		_, err := b.client.Stat(isReadyNamespace, isReadyName)
```

Comment:
I think we need to clarify the goal of the readiness endpoint: to check backends reachable or backend function correctly?

If the goal is to verify backends are reachable, how about just pinging or HTTP check the backend endpoints?

Reply:
The stronger the check better. I've reworked the check to instead download an empty file. Unfortunately, this requires the Kraken user to manually upload an empty file before being able to use the endpoint.